Foundation/NSRunLoop.swift

```diff
@@ -33,14 +33,14 @@ public struct RunLoopMode : RawRepresentable, Equatable, Hashable, Comparable {
     public var hashValue: Int {
         return rawValue.hashValue
     }
-}
 
-public func ==(lhs: RunLoopMode, rhs: RunLoopMode) -> Bool {
-    return lhs.rawValue == rhs.rawValue
-}
+    public static func ==(_ lhs: RunLoopMode, _ rhs: RunLoopMode) -> Bool {
+        return lhs.rawValue == rhs.rawValue
+    }
 
-public func <(lhs: RunLoopMode, rhs: RunLoopMode) -> Bool {
-    return lhs.rawValue < rhs.rawValue
+    public static func <(_ lhs: RunLoopMode, _ rhs: RunLoopMode) -> Bool {
```

Comment:
It seems like a bug, and unless all of our extensible string enums in scl-foundation have this operator I'd prefer to remove it here.

(if you really do want to compare the strings, then using rawValue with `<` seems reasonable to me)